src/brevitas/graph/calibrate.py

```diff
@@ -47,67 +50,36 @@
 BN_LAYERS = (nn.BatchNorm1d, nn.BatchNorm2d, nn.BatchNorm3d)
 
 
-def disable_return_quant_tensor(model):
-    previous_state = {}
-    for module in model.modules():
-        if hasattr(module, 'return_quant_tensor'):
-            previous_state[module] = module.return_quant_tensor
-            module.return_quant_tensor = False
-    return previous_state
+def unpack_input(inp: Union[tuple, QuantTensor]) -> torch.Tensor:
```

Comment:
We already have this in brevitas.quant_tensor, it's called _unpack_quant_tensor

If that doesn't work with tuple, it might be worth extending it (with the assumption that a tuple is always a decayed QT)